.circleci/config.yml

```diff
@@ -49,10 +49,11 @@ jobs:
             - node_modules
 
       # prepare the database
+      - run: cp .env.testing .env
```

Comment:
it's testing env why we add it ?

Reply:
substitute the normal production env and circleci env every one for it's default configuration